RocketChat/Rocket.Chat
# fix: Cron job for clearing OEmbed cache isn't working

## Proposed changes (including videos or screenshots)

 - Fix error running cron job for clearing OEmbed cache (we shouldn't check for the `uid` in this case);
 - Fix `data.params undefined` error when clearing OEmbed cache (eg with the "Clear" button in the settings).

## Issue(s)

Fixes #29434

## Steps to test or reproduce

Send some messages containing links with previews, then go to **Workspace > Settings > Message** and check the value for the **Embed Cache Expiration Days** setting. A cron job to erase all OEmbed cache registries older than `Embed Cache Expiration` days should be erased from the database (there should be no document in the `rocketchat_oembed_cache collection` with an `updatedAt` field older than defined in the setting).
The cron job runs everydat at 2:24am (this can be changed in [this line](https://github.com/RocketChat/Rocket.Chat/pull/31336/files#diff-21161a95fbf3394faa4f2af4add5e90d8eeeb279694d3107c86f834c41059e99R6) for test purposes -- use it as `* * * * *` to run every minute, for example).

## Further comments

SUP-425
